packages/salesforcedx-apex-replay-debugger/src/adapter/apexReplayDebug.ts

```diff
@@ -222,13 +221,41 @@ export class ApexReplayDebug extends LoggingDebugSession {
     args: DebugProtocol.InitializeRequestArguments
   ): void {
     this.initializedResponse = response;
-    this.sendEvent(new Event(GET_LINE_BREAKPOINT_INFO_EVENT));
+    this.initializedResponse.body = {
+      supportsConfigurationDoneRequest: true,
+      supportsCompletionsRequest: false,
+      supportsConditionalBreakpoints: true,
+      supportsDelayedStackTraceLoading: false,
+      supportsEvaluateForHovers: false,
+      supportsExceptionInfoRequest: false,
+      supportsExceptionOptions: false,
+      supportsFunctionBreakpoints: false,
+      supportsHitConditionalBreakpoints: false,
+      supportsLoadedSourcesRequest: false,
+      supportsRestartFrame: false,
+      supportsSetVariable: false,
+      supportsStepBack: false,
+      supportsStepInTargetsRequest: false
+    };
+    this.initializedResponse.success = true;
+    this.sendResponse(this.initializedResponse);
   }
 
   public async launchRequest(
     response: DebugProtocol.LaunchResponse,
     args: LaunchRequestArguments
   ): Promise<void> {
+    let lineBreakpointInfoAvailable = false;
+    if (args && args.lineBreakpointInfo) {
```

Comment:
Get the lineBreakpointInfo from the `args` sent to launchRequest and remove it from it to avoid polluting the rest of the methods using `args`.